recipe/meta.yaml

```diff
@@ -24,13 +24,14 @@ requirements:
     - wheel
     - setuptools
   run:
-    - python
+    - python >=3.7
     - requests >=2.16.2
     - six >=1.12.0
     - stone >=2.*
 
 test:
   requires:
+    - python <3.10
```

Comment:
This should not be required for not noarch packages.

Reply:
If we'll rebuild it as **noarch** we can keep it as it is